dali/base/dadfs.cpp

```diff
@@ -3215,6 +3215,22 @@ class CDistributedFileBase : implements INTERFACE, public CInterface
     virtual bool getAccessedTime(CDateTime &dt) = 0;                            // get date and time last accessed (returns false if not set)
     virtual void setAccessedTime(const CDateTime &dt) = 0;                      // set date and time last accessed
     virtual bool isExternal() const { return external; }
+
+    virtual bool getExpire(int &expireDays)
+    {
+        expireDays = queryAttributes().getPropInt("@expireDays", -1);
+        return true;
+    }
+
+    virtual bool setExpire(int expireDays)
```

Comment:
not sure point of boolean return value, since it's always true, why not void?

Reply:
Changed